docs/pgx.md

```diff
@@ -0,0 +1,44 @@
+# Google Cloud Spanner PostgreSQL Adapter - pgx Support
+
+PGAdapter has limited support for the [Go pgx driver](https://github.com/jackc/pgx) version 4.15.0
+and higher. 
+
+## Usage
+
+First start PGAdapter:
```

Comment:
do not duplicate this across docs.. rather link to the main README so that there is a variety of ways to "start" adapter

Reply:
I've added this to all specific documents as it is likely that we at least temporarily will need specific options for specific drivers. For pgx we currently do not have that (although it is very likely that it will change). I've tried to use the most likely start option for each tool (e.g. Docker when using pgx, as it is outside the Java ecosystem, fat-jar when using JDBC)